autotest/gdrivers/rraster.py

```diff
@@ -62,10 +62,12 @@ def test_rraster_1(filename='data/rraster/byte_rraster.grd', check_prj=None):
 def test_rraster_1_copy():
 
     filename = '/vsimem/rraster/byte_rraster.grd'
-    gdal.Translate(filename, 'data/rraster/byte_rraster.grd', format='RRASTER')
-    assert not gdal.VSIStatL(filename + '.aux.xml'), 'did not expect .aux.xml'
+#    gdal.Translate(filename, 'data/rraster/byte_rraster.grd', format='RRASTER')
+    gdal.Translate(filename, 'data/rraster/byte_rraster1.grd', format='RRASTER')
+#    assert not gdal.VSIStatL(filename + '.aux.xml'), 'did not expect .aux.xml'
```

Comment:
why is that commented ? because there's now a .aux.xml file generated ? Is it expected / what is its content ?

Reply:
An .aux.xml file was generated when creating the updated version of rraster.grd as rraster1.grd. I deleted it manually, because I do not know why it was created; again, I'm unsure how adding a key could have precipitated the creation of the .aux.xml file.

Reply:
No .aux.xml is created in 3.4.2 from data/byte.tif. None from `gdal_translate -of RRASTER -ot Byte byte.tif ~/tmp/byte.grd` either (run in gdal/autotest/gdrivers/data).